VideoWeb/VideoWeb/ClientApp/src/app/services/video-filter.service.ts

```diff
@@ -102,6 +107,12 @@ export class VideoFilterService {
     }
 
     updateCameraStream(stream: MediaStream) {
+        this._canvasWidth = stream.getVideoTracks()[0].getSettings().width;
```

Comment:
Good call, I missed this!

Reply:
This didn't work as expected. Going to raise this as a separate ticket.